### PR TITLE
Feat/text area component

### PR DESCRIPTION
**Pull request by merewelch214**
### PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

### PR Type

This PR updates the text input component in order to add the option to make it a multiline text area. 

-   [ ] Bugfix
-   [x] Feature
-   [ ] Code style update (formatting, local variables)
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] CI related changes
-   [ ] Performance improves
-   [ ] Adding missing tests
-   [ ] Documentation content changes
-   [ ] Other... Please describe:

## Fixes # (issue)

[PNA-59](https://medlypharmacy.atlassian.net/jira/software/projects/PNA/boards/6?selectedIssue=PNA-59)

### What is the new behavior?

Text input becomes a text area when the multiline prop is set to true

### Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### Additional context

Add any other context or screenshots about the feature pr here.

### CODE REVIEW

**Comment by merewelch214** [comment]
@gmukul01 I believe this is ready for your review. I couldn't get the tests to pass for NavItem, SideNav, Actions, ToggleSwitch or Form with the error: Cannot find module '@medly-components/core' from '_component.tsx_'. Have you seen this error before? Wondering what I'm missing. Thanks!

File: packages/core/src/components/TextField/Styled/TextField.styled.tsx [hunk 1]
```diff
@@ -22,22 +33,34 @@ const styleWithLabel = ({ variant }: StyledProps) => {
...
+    }
+    return getFontStyle({ theme, fontVariant: theme.textField.textVariant[size] });
+};
+
+const setHeight = ({ multiline }: StyledProps) => multiline && `height: 100%`;
+
 export const Input = styled('input')<StyledProps>`
     color: ${({ variant, theme }) => theme.textField[variant].default.textColor};
     width: 100%;
+    ${setHeight};
```
Comment (gmukul01) [inline packages/core/src/components/TextField/Styled/TextField.styled.tsx:48]: You can write this directly over here only.
height: ${({multiline}) => multiline && '100%'}

File: packages/core/src/components/TextField/Styled/TextField.styled.tsx [hunk 2]
```diff
@@ -22,22 +33,34 @@ const styleWithLabel = ({ variant }: StyledProps) => {
...
     text-overflow: ellipsis;
+    resize: none;
     z-index: 1;
+    scroll-padding-top: 10px;
 
     ${({ isLabelPresent, size }) => isLabelPresent && size !== 'S' && styleWithLabel};
     &,
     & ~ ${MaskPlaceholder} {
-        ${({ theme, size }) => getFontStyle({ theme, fontVariant: theme.textField.textVariant[size] })}
+        ${setFontStyle}
```
Comment (gmukul01) [inline packages/core/src/components/TextField/Styled/TextField.styled.tsx:63]: you can write directly over here 
`${({theme, size, multiline }) => getFontStyle({ theme, fontVariant: multiline ? 'body2' : theme.textField.textVariant[size] })}`

File: packages/forms/src/components/Form/__snapshots__/Form.test.tsx.snap [hunk 3]
```diff
@@ -26,7 +26,7 @@ exports[`Form should not render any field if component type is not matched 1`] =
   text-decoration: none;
   -webkit-transition: all 100ms ease-out;
   transition: all 100ms ease-out;
-  padding: 1rem 2.4rem 1.2rem;
+  padding: 1rem 2.4rem 1.2rem 2.4rem;
```
Comment (gmukul01) [inline packages/forms/src/components/Form/__snapshots__/Form.test.tsx.snap:29]: I am just wondering from where this `2.4rem` is coming ?

**Review by gmukul01 — CHANGES_REQUESTED** [review]

File: packages/core/src/components/TextField/Styled/InnerWrapper/InnerWrapper.styled.tsx [hunk 4]
```diff
@@ -91,6 +91,30 @@ const errorStyle = ({ theme: { textField }, variant }: InnerWrapperProps) =>
...
+
+    if (multiline) {
+        if (size === 'S' || variant === 'fusion') {
+            return lineHeightNumber * minRows + smallPadding;
+        } else {
+            return lineHeightNumber * minRows + medPadding;
+        }
+    }
+    return baseHeightNumber;
+};
```
Comment (jaredgiangrasso) [inline packages/core/src/components/TextField/Styled/InnerWrapper/InnerWrapper.styled.tsx:112]: This is great 👍

**Review by jaredgiangrasso — APPROVED** [review]

**Review by gmukul01 — APPROVED** [review]
LGTM

**Pull request merged**